QueAns/tests.py

```diff
@@ -0,0 +1,16 @@
+#couldn't test it
+from django.test import Client
+
+c = Client()
+response = c.post('/upvote/',{'user_id':'prakhar_96','question_id':'1'})
+self.assertEqual(response.status_code, 200)
+
+response = c.post('/downvote',{'user_id':'prakhar_96','question_id':'1',})
+self.assertEqual(response.status_code,200)
+
+response = c.post('/satisfied/',{'user_id':'prakhar_96','question_id':'1'})
+self.assertEqual(response.status_code, 200)
+
+response = c.post('/disatisfied/',{'user_id':'prakhar_96','question_id':'1',})
+self.assertEqual(response.status_code,200)
```

Comment:
These tests aren't exhaustive enough.
https://docs.djangoproject.com/en/2.0/intro/tutorial05/#testing-our-new-view

Reply:
http://softwaretesting-qaqc.blogspot.in/2012/12/test-case.html